datalad/support/third/loris_token_generator.py

```diff
@@ -0,0 +1,23 @@
+import sys
```

Comment:
Please borrow the header from other files, or add a custom one (about license/copyright, if needed)